### PR TITLE
Fixed linting on all base style files

### PR DESCRIPTION
**Pull request by richmccartney**
#### What's this PR do?

Lints all _base files to not error or show warnings
#### Where should the reviewer start?

Pull the branch down and run `npm i`
#### How should this be manually tested?

Run `gulp test` and make sure none of the `_base` prefixed files show a linting error

### CODE REVIEW

File: .sass-lint.yml [hunk 1]
```diff
@@ -33,7 +33,12 @@ rules:
   no-color-keywords: 2
   no-css-comments: 2
   no-debug: 2
-  no-duplicate-properties: 2
+  no-duplicate-properties:
+    - 2
+    -
```
Comment (anthonydillon) [inline .sass-lint.yml]: Extra bullet here.

File: scss/_base_forms.scss [hunk 2]
```diff
@@ -7,10 +7,8 @@
 /// Form element styles
 /// @group Forms
 @mixin maas-forms {
-
-  // All input elements
+  //scss-lint: disable no-vendor-prefixes
```
Comment (anthonydillon) [inline scss/_base_forms.scss]: This should be disabled around the thing its disabling so we dont leave it and get confused.

File: scss/_base_forms.scss [hunk 3]
```diff
@@ -61,63 +59,60 @@
...
-      background-image: none;
       background-color: $color-transparent;
+      background-image: none;
+      color: $color-mid-dark;
     }
 
     // Removes firefox dotted outline focusing
-    &:-moz-focusring {
-      color: transparent;
+    &:-focusring {
```
Comment (anthonydillon) [inline scss/_base_forms.scss]: focusring is just a moz pseudo element.

File: scss/_base_forms.scss [hunk 4]
```diff
@@ -61,63 +59,60 @@
...
+      color: $color-transparent;
       text-shadow: 0 0 0 $black;
     }
 
-    &::-ms-expand {
+    &::-expand {
```
Comment (anthonydillon) [inline scss/_base_forms.scss]: This is just an `ms` pseudo element

**Review by anthonydillon — CHANGES_REQUESTED** [review]
QA good with a few comment in code.

**Comment by richmccartney** [comment]
@anthonydillon if you can take a second look at this please.

**Review by anthonydillon — APPROVED** [review]

**Pull request merged**